Fjerner weight 400 og størrelsene xs og 2xl i ffe-symbols

HeleneKassandra opened the pull request:
<!-- Gi saken en oppsummerende tittel ovenfor. -->
## Beskrivelse
Fjerner 400 som font-weight, både i ikon-font import og alle stylinger og referanser til det i komponenten.
Fjerner også størrelsene xs og 2xl
<!-- Detaljert beskrivelse av endringene dine. Skjermskudd er lov. -->

## Motivasjon og kontekst
I overgangen til SVG-implementering så fjernet vi 2 weights og 2 størrelser for å minske antall ikoner. 

<!-- Hvorfor trengs denne endringen, og hva løser den? -->

## Testing

<!-- Hvordan har du har testet endringene dine? Ta gjerne med systeminfo o.l -->

<!--

Helt til slutt, har du ..

.. lest retningslinjene våre for bidrag?
.. tatt en siste sjekk for kode og skrivefeil, debug informasjon o.l?
.. kjørt og eventuelt oppdatert testene?
.. kommentert og dokumentert det som trengs?
.. knyttet denne opp til relaterte issues?

-->

HeleneKassandra committed:
fix(component-overview): fjerner 2 størrelser og weights i symbols eksempler

andrea-sb1 reviewed (changes requested):
Takk for at du oppdaterte eksemplene 🙏 Mulighet til å endre weight fra 400 til 300 i eksemplene «Symbol», «Symbol-allsizes», «Symbol-arialabel» og «Symbol-filled»? 😊

HeleneKassandra committed:
fix(ffe-symbols-react): fjern weight 400 og størrelsene xs og 2xl
BREAKING CHANGE: Fjerner 400 som weight og størrelsene xs og 2xl fra Symbol-komponenten.

HeleneKassandra commented:
> Takk for at du oppdaterte eksemplene 🙏 Mulighet til å endre weight fra 400 til 300 i eksemplene «Symbol», «Symbol-allsizes», «Symbol-arialabel» og «Symbol-filled»? 😊

Fikset nå! Det var at defaultverdien i symbols komponenten var 400. Endret den til 300 nå!

HeleneKassandra committed:
fix(ffe-symbols): fjern weight 400 og størrelsene xs og 2xl
BREAKING CHANGE: fjerner fontene for weight 400 og styling for xs og 2xl
